.azure-pipelines/common-templates/download-openapi-docs.yml

```diff
@@ -10,6 +10,8 @@ parameters:
     displayName: Build Agent
   - name: SkipForceRefresh
     type: string
+  - name: SkipSingularizeOperationIds
```

Comment:
Remove the unnecessary `SkipSingularizeOperationIds` changes in the pipeline build scripts. DevX API will skip singularization of OperationIds by default. Singularization of operation ids will be covered by https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/1899.